05 type conv/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"reflect"
+)
+
+type counter int
+
+func main() {
+
+	var d counter
+	d = 0
```

Comment:
OK, noted. Was just following instructions step by step . I will try and
step up like a PRO.😃




Ayo Adesokan






On Fri, Jul 1, 2016 at 8:13 AM, Ewetumo Alexander <notifications@github.com>
wrote:

> In 05 type conv/main.go
> <https://github.com/adesokanayo/goclass/pull/2#discussion_r69258715>:
>
> > @@ -0,0 +1,25 @@
> > +package main
> > +
> > +import (
> > +	"fmt"
> > +	"reflect"
> > +)
> > +
> > +type counter int
> > +
> > +func main() {
> > +
> > +	var d counter
> > +	d = 0
>
> Why declare a variable like this then now give it value latter right below
> its, next time just use this:
>
> d := 0
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/adesokanayo/goclass/pull/2/files/64afaf8be15368355f3e4072898325a7402b62c2..e40c4dd21170904532209009a11832df5f6fac3d#r69258715>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFINpm6Qtw0_eSK_wMfnBoAMb6kjzpu6ks5qRL4FgaJpZM4JBmwX>
> .
>